src/js/client/handlebars.js

```diff
@@ -0,0 +1,172 @@
+// A client-side module that provides various template handling capabilities, and which wires in any
+// child components with the grade `gpii.templates.hb.helper` as handlebars helpers.
+//
+// Requires Handlebars.js and Pagedown (for markdown rendering)
+
+/* global fluid, jQuery, Handlebars */
+(function ($) {
+    "use strict";
+    var gpii = fluid.registerNamespace("gpii");
+    fluid.registerNamespace("gpii.templates.hb.client");
+
+    gpii.templates.hb.client.addHelpers = function (that) {
+        if (Handlebars) {
+            if (that.options.components) {
```

Comment:
As with express itself, iteration through components isn't a completely reliable way to detect subcomponents - should use options distribution instead.